src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -104,6 +104,48 @@ public function init_form_fields()
                 $this->paymentConfirmationAfterCoupleOfDays()
         );
 
+    }
+
+    public function init_settings()
+    {
+        parent::init_settings();
+        if(is_admin()){
+            global $current_section;
+            wp_register_script(
+                    'mollie_wc_gateway_settings',
+                    Mollie_WC_Plugin::getPluginUrl(
+                            '/public/js/gatewaySettings.min.js'
+                    ),
+                    ['underscore', 'jquery'],
+                    Mollie_WC_Plugin::PLUGIN_VERSION
+            );
+
+            wp_enqueue_script('mollie_wc_gateway_settings');
+            wp_enqueue_style('mollie-gateway-icons');
+            $settingsName = "{$current_section}_settings";
+            $gatewaySettings = get_option($settingsName, false);
```

Comment:
I guess you don't really need this as you already have it in `$this->settings`. But this is not a problem that needs to be fixed, just a note.